### PR TITLE
Apply range validation to MCI/RSI questionnaires

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Validation changes as specified in https://trello.com/c/0Z2bqo7U/1533-apply-item-must-be-or-to-total-to-mci-rsi

### How to review 
Are all of the changes correct.

### CODE REVIEW

File: data/en/1_0102.json [hunk 1]
```diff
@@ -208,7 +208,10 @@
                         "q_code": "21",
                         "type": "Currency",
                         "currency": "GBP",
-                        "decimal_places": 2
+                        "decimal_places": 2,
+                        "max_value": {
```
Comment (charlotte-hirst) [inline data/en/1_0102.json:212]: Which error message will it display?

Comment (jonnyshaw89) [reply data/en/1_0102.json:212]: `Enter a number less than or equal to £100.00.`

Comment (shahi645) [reply data/en/1_0102.json:212]: @charlotte-hirst  Once updated with master will read `Enter an answer less than or equal to £100.00.`

**Review by shahi645 — APPROVED** [review]
Matches spec file

**Review by charlotte-hirst — APPROVED** [review]

**Pull request merged**